validator/validator-main.protoascii

```diff
@@ -5321,6 +5320,39 @@ tags: {
   }
 }
 
+tags: {
+  html_format: AMP
+  enabled_by: "transformed"
+  tag_name: "IMG"
+  spec_name: 'I-AMPHTML-INTRINSIC-SIZER'
```

Comment:
spec_names need some rework at some point, but in the mean time, I'd like to make sure that they all include the actual tag name somewhere. So IMG + whatever

Reply:
done